.gitignore

```diff
@@ -1 +1,2 @@
 .DS_Store
+docker-compose.yml
```

Comment:
Adding `.docker/` here could be a good idea, so it won't show as unstaged in git.